test.sh

```diff
@@ -8,24 +11,13 @@ print_in_cyan() {
     printf "\x1B[36m%b\x1B[0m\n" "$*"
 }
 
-PYTHON_VERSIONS="3.8  3.9"
-TEST_SQLALCHEMY=true
-TEST_DJANGO=true
+PYTHON_VERSIONS=('3.8' '3.9')
```

Comment:
Suggest testing on 3.7 and 3.10 aka the oldest supported and also latest release. If something breaks you can narrow it down from there. Skip 3.8 and 3.9 because you can induce that if it passes on both 3.7 and 3.10 it will pass on those as well

Reply:
3.10 is coming in a future release. There are some issues with 3.6 and 3.7 because internal python changes changed how the stacktraces are computed, so the snapshots break. We're thinking about a way to solve it.